landuse_classification/anaconda-project.yml

```
@@ -4,6 +4,7 @@ description: Image classification using the UC Merced Land Use Dataset
 examples_config:
   maintainers:
   - jbednar
+  - philippjfr
```

Philipp did way more on this than I did:
```suggestion
  - philippjfr
  - jbednar
```